### ISSUE TITLE
Bootstrap table css not included with _content/Radzen.Blazor/css/default.css

### ISSUE THREAD

**Issue by Matthewsre** [radzenhq/radzen-blazor#217]
Bootstrap table css not included with _content/Radzen.Blazor/css/default.css

**To Reproduce**

1. Include default.css file which should include boostrap according to the [Get Started](https://blazor.radzen.com/get-started) page
> Alternatively you can include <link rel="stylesheet" href="_content/Radzen.Blazor/css/default.css"> which embeds Bootstrap.

`<link rel="stylesheet" href="_content/Radzen.Blazor/css/default.css">`

2. Add the following to your page (Sample from bootstrap):

```
<table class="table table-striped">
  <thead>
    <tr>
      <th scope="col">#</th>
      <th scope="col">First</th>
      <th scope="col">Last</th>
      <th scope="col">Handle</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <th scope="row">1</th>
      <td>Mark</td>
      <td>Otto</td>
      <td>@mdo</td>
    </tr>
    <tr>
      <th scope="row">2</th>
      <td>Jacob</td>
      <td>Thornton</td>
      <td>@fat</td>
    </tr>
    <tr>
      <th scope="row">3</th>
      <td>Larry</td>
      <td>the Bird</td>
      <td>@twitter</td>
    </tr>
  </tbody>
</table>
```

3. You can see the following result is not rendering bootstrap tables correctly:

![image](https://user-images.githubusercontent.com/2955659/133528296-f7bff005-a0ae-498c-b03c-6d0f2ab85b11.png)

**Comment by akorchev** [comment]
This is by design - the Radzen themes do not include the complete Bootstrap CSS as they are not using most of it. Feel free to include Bootstrap in your app and use the base version of the theme as mentioned in the documentation.

**Comment by Matthewsre** [comment]
> This is by design - the Radzen themes do not include the complete Bootstrap CSS as they are not using most of it. Feel free to include Bootstrap in your app and use the base version of the theme as mentioned in the documentation.

@akorchev,

Thank you for clarifying and I resolved the issue by including complete Bootstrap. 

May I suggest updating the documentation to reflect what Bootstrap capabilities are included with the default.css? 

My assumption was that "embeds Bootstrap" meant includes full Bootstrap and it took me a while to figure out why tables weren't working. The clarity could help prevent others from making the same assumptions I did and save them some trouble.
